src/pages/activities/[page].tsx

```diff
@@ -0,0 +1,37 @@
+import { getActivityByName, getAllActivities } from "../../libs/activitiesAPI";
+
+const Page = (page: JSON) => {
+  return JSON.stringify(page);
+};
+
+export default Page;
+
+type Params = {
+  params: {
+    page: string;
+  };
+};
+
+export async function getStaticProps({ params }: Params) {
```

Comment:
This type is not needed use the types found in https://nextjs.org/docs/basic-features/typescript